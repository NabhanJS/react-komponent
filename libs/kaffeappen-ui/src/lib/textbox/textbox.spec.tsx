import { render } from '@testing-library/react';
import Textbox from './textbox';

describe("Textbox", () => {
    it("should test the Textbox", () => {
        const { baseElement } = render(< Textbox placeholder={'Username'} type={"Username"}/>)
        expect(baseElement).toBeTruthy()
    })

    it("should match the snapshot", () => {
        const { baseElement } = render(< Textbox placeholder={'Username'} type={"Username"}/>)
        expect(baseElement).toMatchSnapshot()
    })
})