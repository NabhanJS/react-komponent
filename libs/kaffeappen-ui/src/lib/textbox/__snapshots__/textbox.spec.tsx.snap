// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Textbox should match the snapshot 1`] = `
<body>
  <div>
    <div>
      <div>
        <input
          placeholder="Username"
          type="text"
        />
      </div>
    </div>
  </div>
</body>
`;
